src/index.ts

```diff
@@ -314,6 +316,7 @@ export class JupiterOneClient {
     this.apiUrl = dev
```

Comment:
That thought definitely crossed my mind, but since this is supplied via environment variables instead of a CLI flag I wasn't sure if that'd be helpful or annoying.  Like if 99% of the time you wanted to just run with the dev flag enabled and then one time target another system, it might be annoying to have to go unset/reset your environment variables back and forth just to set the apiBaseUrl for a brief period.  Certainly open to challenge on that though.